openedx/core/djangoapps/programs/tests/mixins.py

```diff
@@ -50,176 +51,48 @@ class ProgramsDataMixin(object):
-            },
-            {
-                'id': 3,
-                'name': PROGRAM_NAMES[2],
-                'subtitle': 'A third program used for testing purposes',
-                'category': 'xseries',
-                'status': 'unpublished',
-                'marketing_slug': '{}_test_url'.format(PROGRAM_NAMES[2].replace(' ', '_')),
-                'organizations': [
-                    {
-                        'display_name': 'Test Organization B',
-                        'key': 'organization-b'
-                    }
-                ],
-                'course_codes': [
-                    {
-                        'display_name': 'Test Course D',
-                        'key': 'course-d',
-                        'organization': {
-                            'display_name': 'Test Organization B',
-                            'key': 'organization-b'
-                        },
-                        'run_modes': [
-                            {
-                                'course_key': COURSE_KEYS[7],
-                                'mode_slug': 'verified',
-                                'sku': '',
-                                'start_date': '2015-11-05T07:39:02.791741Z',
-                                'run_key': 'winter'
-                            }
-                        ]
-                    }
-                ],
-                'created': '2015-10-26T19:59:03.064000Z',
-                'modified': '2015-10-26T19:59:18.536000Z'
-            }
+            factories.Program(
+                id=1,
+                name=PROGRAM_NAMES[0],
+                organizations=[factories.Organization()],
+                course_codes=[
+                    factories.CourseCode(run_modes=[
+                        factories.RunMode(course_key=COURSE_KEYS[0]),
+                        factories.RunMode(course_key=COURSE_KEYS[1]),
+                    ]),
+                    factories.CourseCode(run_modes=[
+                        factories.RunMode(course_key=COURSE_KEYS[2]),
+                        factories.RunMode(course_key=COURSE_KEYS[3]),
+                    ]),
+                ]
+            ),
+            factories.Program(
+                id=2,
+                name=PROGRAM_NAMES[1],
+                organizations=[factories.Organization()],
+                course_codes=[
+                    factories.CourseCode(run_modes=[
+                        factories.RunMode(course_key=COURSE_KEYS[4]),
+                        factories.RunMode(course_key=COURSE_KEYS[5]),
+                    ]),
+                    factories.CourseCode(run_modes=[
+                        factories.RunMode(course_key=COURSE_KEYS[6]),
+                        factories.RunMode(course_key=COURSE_KEYS[7]),
+                    ]),
+                ]
+            ),
+            factories.Program(
+                id=3,
+                name=PROGRAM_NAMES[2],
+                organizations=[factories.Organization()],
```

Comment:
Prior to this change, this program (id=3) had the same organization as program 2.  However, I'm fine with leaving it this way if the tests still pass.